dev/EnvironmentManager/ChangeTrackerHelper/ChangeTrackerHelper/ChangeTrackerHelper.h

```diff
@@ -0,0 +1,56 @@
+﻿#pragma once
+#include <windows.h>
+#include <appmodel.h>
+
+enum AreChangesTracked
+{
+    YES,
+    NO,
+    NOT_CHECKED
+};
+
+AreChangesTracked m_AreChangesTracked{NOT_CHECKED};
```

Comment:
A global variable? In a header file? Not a good idea e.g. if this header is #include'd in 2+ files you get multiple instances and potential redefinition errors at link time or worse, you don't and you get weird runtime behavior. Actual outcome depends, but either way BadMojo(TM).

Reply:
This file is removed.